README.md

```diff
@@ -2,4 +2,52 @@
 
 Northeastern Electric Racing Project Management Dashboard
 
-All issues and new features are handled in the Github Repository Issues tab
\ No newline at end of file
+### Some helpful steps:
+- Installing the project for the first time:
+	- Make a folder somewhere (note this is not the NER dashboard parent directory. That directory will be placed inside the one you create)
+	- Open a terminal in the folder
+	- Type `git clone https://github.com/Northeastern-Electric-Racing/NER-PM-Dashboard`
+	- Now enter the directory by typing `cd NER-PM-Dashboard`
+
+- To create an issue (or suggest features):
+	- Open the github URL online
+	- Go to the issues tab and click add new issue
+	- Give it a name that gives the reader an idea of whats going on. Save all detailed info for the description
+	- On the right hand coloumn add labels to your issue to categorise it meaningfully
+	- Add it to board_1 **To Do** initially.
+	- When you are ready to work on the issue, assign yourself to it and then move it to the **In Progress** board
+
+- Creating branches:
+	- In the NER-PM folder on your computer open a terminal window
+	- Type `git branch __name of branch__`
+	- By convention, your branch should contain your username to make it easier to track development progress and pull requests
+	- They type `git push` to push your branch to the repo
+	- If you have been working on your branch for a while and the main repo has changed, type `git pull --rebase` to sync those changes to your branch
+	- If you close the terminal window and reopen it, you will be put back onto the master branch by default. This will give errors when trying to push so use `git switch __name of branch__` to switch back to your branch
```

Comment:
```suggestion
	- If you close the terminal window and reopen it, you will be put back onto the master branch by default. This will give errors when trying to push so use `git switch [name of branch]` to switch back to your branch
```